### ISSUE TITLE
Keyboard navigation inside the PLIX iframe

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#6015]
### Observed behavior

Navigating inside the PLIX `iframe` by keyboard, especially trying to catch the `Challenge me` button in focus is very frustrating and ultimately unsuccessful, as it never gains the visual focus indicator, and I was unable to even guess when to press Enter. 

Outer scrollbar also seems unnecessary on both browsers.

#### Firefox

PLIX `iframe` does display focus visually, so at least there's indication that scrolling might be possible, but the elements inside lack visual indicators when in focus.
![plix](https://user-images.githubusercontent.com/1457929/67999805-ff7c3900-fc5d-11e9-8607-004144fcb5f4.gif)

#### Chrome

PLIX `iframe` does not display visual focus indicator, so it's unclear when the user could use arrow keys to scroll.
![focus-PLIX](https://user-images.githubusercontent.com/1457929/68065204-0d958c80-fd26-11e9-9538-b25e8a25b10e.gif)

Small arrow icon alongside the license type is curious case, as it seems to lack the visual indicator when one uses <kbd>TAB</kbd> key going down, but it displays the focus indicator when one uses <kbd>Shift+TAB</kbd> to go back... :confused: 

Is this is something to be addressed on the content level? If yes, I'll file an issue on the sushi chef repo...

### Expected behavior
Being able to navigate and activate elements inside `iframe`.

### User-facing consequences
Inability to interact with the resource when using keyboard.

### Errors and logs
…

### Steps to reproduce
Try to navigate inside by keyboard.

### Context
 * Kolibri version: current `develop`
 * Operating system: Ubuntu 18.04
 * Browser: both FF & Chrome

**Comment by rtibbles** [comment]
Should this be opened as an issue on the CK12 chef repo, rather than on Kolibri? Sounds like the issues are only with the PLIX elements here - unless there could be some general purpose accessibility scaffolding for HTML5 apps we can do in Kolibri?

**Comment by kollivier** [comment]
Yeah, this seems likely to be a CK12 content issue to me as well.
